### PR TITLE
Fix initial keyboard focus problems

### PR DESCRIPTION
**Pull request by jherico**
Attempting to solve https://app.asana.com/0/27650181942747/83176475832393

Needs testing by someone who can reproduce the bug reliably, which I cannot.

### CODE REVIEW

File: libraries/gl/src/gl/OffscreenQmlSurface.cpp [hunk 1]
```diff
@@ -610,6 +610,8 @@ bool OffscreenQmlSurface::isPaused() const {
 
 void OffscreenQmlSurface::setProxyWindow(QWindow* window) {
     _renderer->_renderControl->_renderWindow = window;
+    window->requestActivate();
```
Comment (ZappoMan) [inline libraries/gl/src/gl/OffscreenQmlSurface.cpp]: aren't these two lines totally redundant?

`_renderer->_renderControl->_renderWindow` is window... form line 612
so you're calling `window->requestActivate()` twice.

Comment (jherico) [inline libraries/gl/src/gl/OffscreenQmlSurface.cpp]: Whoops... you're right... the second line should be `_renderer->_quickWindow->requestActivate();`  will fix.

**Comment by hifi-gustavo** [comment]
Build console output for commit: 87e169cb891f2c1e57ee7a278bd66cd6eaed846a available at: https://gist.github.com/0b5668a3883dbc4cd980<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR6905.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR6905.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 46b4ce9f0578b51b48276754dcf2c677fe8a9c5f available at: https://gist.github.com/0b5668a3883dbc4cd980<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR6905.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR6905.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: b29e71cb85b593922d68a2ad7a33e81c5d144f73 available at: https://gist.github.com/0b5668a3883dbc4cd980<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR6905.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR6905.dmg

**Comment by jherico** [comment]
@ctrlaltdavid has built from this branch and says it corrects the problem, which is reliably reproducible on his system.

**Comment by hifi-gustavo** [comment]
Build console output for commit: 89b8eb2efa754eb9c04d976a31ec82316be8b6e5 available at: https://gist.github.com/0b5668a3883dbc4cd980<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR6905.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR6905.dmg

**Comment by ctrlaltdavid** [comment]
Downloaded and tested PR build. Successfully get focus in address bar and "filter" field of Running Scripts dialog without having to switch applications.

**Pull request merged**